HomeTask3/getMiddle.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Calculates and returns the middle part of string
+ * @param {string} str the string from which you want the middle characters
+ * 
+ * @returns {string} middle string
+ * @example getMiddle("test") returns "es", getMiddle("tests") returns "s"
+ */
+function getMiddle(str) {
+    let strlen = str.length;
```

Comment:
Use const instead of let as you are not reassigning a new value to the variables

Reply:
done.